playground/mocks/data/idp/idx/authenticator-expired-custom-password.json

```diff
@@ -38,7 +38,7 @@
           }
         ],
         "accepts": "application/json; okta-version=1.0.0",
-        "customExpiredPasswordName": "expiredpasswordOIE",
+        "customExpiredPasswordName": "Password reset website name",
```

Comment:
changed mock value here to match `customExpiredPasswordName` in Custom password expiry warning mock file to make it more consistent.  Also so only one spec test object method is needed to select the button